.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
`package-lock.json` is intended to be checked into source control.
It describes a single representation of a dependency tree, guarantees exact same version of every package.